### ISSUE TITLE
Cannot load pretrained model

### ISSUE THREAD

**Issue by SM1991CODES** [cardwing/Codes-for-PVKD#37]
Hi,

Thanks for your nice work!!

I was trying to evaluate your model on the validation set of SemanticKITTI for semantic segmentation and I have the following structure:
![image](https://user-images.githubusercontent.com/69435296/224835217-cc2532bd-78c2-4f8d-b594-5fd6a810920c.png)

However, it fails with following error for both models:
![image](https://user-images.githubusercontent.com/69435296/224835334-9470cb37-bd8b-4201-8c67-850b33819ab4.png)

Please help me figure this out.

Also another question - are these trained models (in the google drive) already finetuned on validation sequence 08? I think if yes then I need to retrain the model from scratch for fair evaluation on validation set. What would you suggest?

Do you have figures for iou on validation set for semantic segmentation?

Best Regards
Sambit
